lib/runtime-c-api/src/import/mod.rs

```diff
@@ -661,6 +668,24 @@ pub unsafe extern "C" fn wasmer_import_func_new(
     Box::into_raw(export) as *mut wasmer_import_func_t
 }
 
+/// Hello
+#[no_mangle]
+#[allow(clippy::cast_ptr_alignment)]
+pub unsafe extern "C" fn wasmer_import_trap(
+    ctx: *const wasmer_instance_context_t,
+    error_message: *const c_char,
+) -> c_uint {
+    let ctx = &*(ctx as *const Ctx);
+    let error_message = CStr::from_ptr(error_message).to_str().unwrap();
+
+    (&*ctx.module)
```

Comment:
technically this can trigger UB too, if the pointer is aligned to Rust's liking or if it's null/invalid (but this may be a larger issue in the C API); not sure there's as much we can do here, though checking for null is something